technical-reports/format/design-token.md

```diff
@@ -23,6 +23,35 @@ The example above therefore defines 1 design token with the following properties
 
 Name and value are both **required**.
 
+Token names are case-sensitive, so the following example with 2 tokens in the same group whose names only differ in case is valid:
+
+<aside class="example">
+
+```json
+{
+  "font-size": { "value": "3rem" },
+  "FONT-SIZE": { "value": "16px" }
+}
+```
+
+</aside>
+
+However, some tools may need to transform names when exporting to other languages or displaying names to the user, so having token names that differ only in case is likely to cause identical and undesirable duplicates in the output. For example, an export tool that converts these tokens to SASS code may generate problematic output like this:
```

Comment:
```suggestion
However, some tools may need to transform names when exporting to other languages or displaying names to the user, so having token names that differ only in case is likely to cause identical and undesirable duplicates in the output. For example, an export tool that converts these tokens to Sass code may generate problematic output like this:
```

Reply:
Good spot. Sorry! After all these years of using Sass, I really ought to know better. Will push a fix in a moment.